san/query.py

```diff
@@ -2,15 +2,21 @@
 from san.error import SanError
 
 
-def get_gql_query(idx, dataset, **kwargs):
-    query, slug = parse_dataset(dataset)
+def get_gql_query(idx, identifier, **kwargs):
+  query, _separator, slug = identifier.partition("/")
+  if slug == '':
```

Comment:
btw, this is not guarding against passing something like `daily_active_addresses/`. Maybe the first case should be `if separator != '' and slug == '': <error>`